_data/accessibility-tests/typography.yml

```diff
@@ -0,0 +1,32 @@
+title: Typography
+component_status: pass
+test_items:
+# General tests
+  - summary: Text meets color contrast requirements.
+    summary_additional: |
+      When you view text on a page and use ANDI or color contrast analyzer to look at the hex colors,
+      the font color meets WCAG color contrast of 4.5:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.6.1
+    wcag_criterion: 1.4.3
+  - summary: Headings are present, appropriate and follow logical order.
+    summary_additional: |
+      When you view text content on a page,
+      headings summarize the content under them and are uniquely styled.
+      `H1`-`H4` levels are nested in a logical order (e.g., levels don't get skipped).
+    test_status: conditional
+    test_type: general
+    version_tested: 3.6.1
+    wcag_criterion: 2.4.6
+# Zoom/screen magnification tests
+  - summary: Text can be resized without loss of information.
+    summary_additional: |
+      When you zoom to 200%,
+      the text format stays consistent, and words don't get obscured.
```

Comment:
> [!note]
> I changed the intro do this to be about zoom since we don't offer instructions for resizing text yet. 
```suggestion
      When you zoom to 200%,
      the text format stays consistent, and words don't get obscured.
```